annif/cli.py

```diff
@@ -123,6 +123,28 @@ def run_show_project(project_id):
     click.echo(template.format('Access:', proj.access.name))
 
 
+@cli.command('init')
+@click.argument('project_id')
+@common_options
+def run_init_project(project_id):
+    """
+    Remove the data files of a project.
```

Comment:
Maybe a nitpick, but I'd change this to `Initialize the project to its original, untrained state.` or something like that - it happens to involve clearing the directory in the current implementation, but this could change later.